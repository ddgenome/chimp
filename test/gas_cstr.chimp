## gas-phase only mechanism
mechanism "gas.mech"
## parameter input
parameter "gas.par"
## gas-phase task
task "gas_cstr.task"
