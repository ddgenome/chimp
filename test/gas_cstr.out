# gas_cstr
# x	A	B	C	D	flow	steps
0.000000e+00	1.000000e+05	1.000000e+05	0.000000e+00	0.000000e+00	1.000000e-05	0
1.000020e+00	9.788747e+04	9.788747e+04	4.212179e+03	1.287448e+01	4.732558e-06	101888
2.000029e+00	9.594649e+04	9.594649e+04	8.060004e+03	4.702192e+01	5.469474e-06	195437
3.000001e+00	9.418054e+04	9.418054e+04	1.153927e+04	9.964296e+01	-1.589013e-05	280799
4.000000e+00	9.254649e+04	9.254649e+04	1.473453e+04	1.724837e+02	-7.130793e-05	360293
5.000017e+00	9.107573e+04	9.107573e+04	1.758776e+04	2.607706e+02	3.210034e-06	432621
6.000017e+00	8.972841e+04	8.972841e+04	2.017751e+04	3.656734e+02	2.335222e-06	499785
7.000020e+00	8.851632e+04	8.851632e+04	2.248668e+04	4.806769e+02	4.668402e-06	561263
8.000024e+00	8.740023e+04	8.740023e+04	2.459102e+04	6.085137e+02	6.311348e-06	618942
9.000045e+00	8.638737e+04	8.638737e+04	2.648107e+04	7.442026e+02	7.475720e-06	672476
1.000002e+01	8.547391e+04	8.547391e+04	2.815792e+04	8.942655e+02	6.904472e-06	722255
